core/loadpoint.go

```diff
@@ -927,6 +1009,32 @@ func (lp *Loadpoint) updateChargerStatus() error {
 	return nil
 }
 
+var _ Consumer = (*Loadpoint)(nil)
+
+// implements interface Consumer, redirects to effectiveCurrent
+// TODO check order- if we have currents we should use them
+func (lp *Loadpoint) MaxPhasesCurrent() (float64, error) {
+	// using effective current in use.
+	// potential issue: slow LP might cause interference or short overload
+	if !lp.charging() {
+		return 0, nil
+	}
+
+	if lp.chargeCurrents != nil {
```

Comment:
These might be outdated values in case of error. make sure `lp.chargeCurrents` is wiped when read error occurs.